tests/flink-it/src/test/scala/org/apache/celeborn/tests/flink/WordCountTest.scala

```diff
@@ -65,13 +66,22 @@ abstract class WordCountTestBase extends AnyFunSuite with Logging with MiniClust
   }
 
   test(getClass.getName + ": celeborn flink integration test - word count") {
+    testWordCount(FallbackPolicy.AUTO)
+  }
+
+  test(getClass.getName + ": celeborn flink integration test with fallback - word count") {
+    testWordCount(FallbackPolicy.ALWAYS)
```

Comment:
Could we verify if Flink uses its built-in shuffle when the fallback.policy is set to FallbackPolicy.ALWAYS?